src/components/TodoModal/TodoModal.tsx

```diff
@@ -2,11 +1,38 @@
+import React, { useState, useEffect } from 'react';
+import cn from 'classnames';
+
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  onClose: () => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, onClose }) => {
+  const [user, setUser] = useState<User | null>(null);
```

Comment:
Initializing the state with props is not recommended. This is because it can lead to bugs and inconsistencies in the state. Instead, you should use the props directly in the component or use them to set the state inside useEffect.